### DISCUSSION TITLE
Please add the full path when linking documents

### DISCUSSION THREAD

**Discussion by BRogers80** [outline/outline#3023] [Ideas]
Dear team,

is it possible to add the full path of the document when you can choose while linking?
We have several documents in a different structure with the same name. It is not possible to pick the right one as you only see the document name.
Can you include the full path or the two parent folders in the menu like this:
![grafik](https://user-images.githubusercontent.com/73225781/151403141-6ad09257-8993-4e61-b336-98b7fe47cc2b.png)

Thank you very much

Florian

**Comment by tommoor** [comment]
Yes, fixed [one hour ago](https://github.com/outline/outline/commit/16277d1074b5c70afc88c5d1e15e243f96f238a0) – will be deployed to getoutline.com later today and in the next OSS release

**Reply by BRogers80** [reply]
Thank you very much
